docs/config.md

```diff
@@ -207,6 +207,27 @@ The partner registry requires authentication for pulling images.  This can be ac
 docker --config=/var/lib/origin/.docker login -u <registry-user> -p <registry-password> registry.connect.redhat.com
 ```
 
+### Quay Registry
+
+Using the Quay registry will allow you to load APBs that are published to the [CoreOS Quay Registry](https://quay.io/about/).  If an authentication `token` is provided, private repositories (*which the token is configured to access*) will also load as well as any public repositories in the specified `org`.
+
+```yaml
+registry:
+  - name: quay_reg
```

Comment:
This looks like something you could add to the broker-apb if you wish, kind of like we do for dockerhub [here](https://github.com/openshift/ansible-service-broker/blob/master/apb/defaults/main.yml#L58-L63). You could also default it to `false` like we do for helm [here](https://github.com/openshift/ansible-service-broker/blob/master/apb/defaults/main.yml#L79).

Reply:
@djzager I can do this in another PR.  This way, I can include other recently-added-adapters to that list as well.

Reply:
@djzager actually, I've just updated it here.  It doesn't make sense to add the others as default